src/fidesops/ops/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -263,16 +265,21 @@ def _send_verification_code_to_user(
     db: Session, privacy_request: PrivacyRequest, email: Optional[str]
 ) -> None:
     """Generate and cache a verification code, and then email to the user"""
+    EmailConfig.get_configuration(
+        db=db
+    )  # Validates Fidesops is currently configured to send emails
     verification_code: str = generate_id_verification_code()
     privacy_request.cache_identity_verification_code(verification_code)
-    dispatch_email(
-        db=db,
-        action_type=EmailActionType.SUBJECT_IDENTITY_VERIFICATION,
-        to_email=email,
-        email_body_params=SubjectIdentityVerificationBodyParams(
-            verification_code=verification_code,
-            verification_code_ttl_seconds=config.redis.identity_verification_code_ttl_seconds,
-        ),
+    dispatch_email_task.apply_async(
+        queue=EMAIL_QUEUE_NAME,
+        kwargs={
+            "action_type": EmailActionType.SUBJECT_IDENTITY_VERIFICATION,
+            "to_email": email,
+            "email_body_params": SubjectIdentityVerificationBodyParams(
+                verification_code=verification_code,
+                verification_code_ttl_seconds=config.redis.identity_verification_code_ttl_seconds,
+            ).dict(),
+        },
```

Comment:
Verified this email dispatch task worked locally, however, running a privacy request doesn't seem to work anymore.  I didn't think you needed to do this, but do we need to specify privacy requests should run in the celery queue?

Reply:
Pretty sure [this is the cause](https://github.com/ethyca/fidesops/pull/1173/files#r964266235). Since the default queue specified in `celery.toml` is `fidesops_worker` and we're explicitly telling the worker not to process any tasks from that queue!